pallets/thea-staking/src/tests.rs

```diff
@@ -3,6 +3,7 @@ use crate::{
 	session::{Exposure, IndividualExposure, StakingLimits, UnlockChunk},
 	ActiveNetworks, Candidates, CurrentIndex, Error, Hooks, Perbill, Stakers, Stakinglimits,
 };
+use frame_benchmarking as _;
```

Comment:
Same

Reply:
removed benchmarking as mock and test env still not there